modules/core/04-channel/keeper/packet.go

```diff
@@ -472,7 +484,7 @@ func (k Keeper) AcknowledgePacket(
 		ctx, connectionEnd, proofHeight, proof, packet.GetDestPort(), packet.GetDestChannel(),
 		packet.GetSequence(), acknowledgement,
 	); err != nil {
-		return sdkerrors.Wrap(err, "packet acknowledgement verification failed")
```

Comment:
all the verify functions indicate what type of verification was done. This is redundant